packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -115,38 +133,108 @@ const renderLabel = (
 const MenuItem = styled.div<MenuItemProps>`
   width: 100%;
   cursor: pointer;
-  color: ${customProperties.colorSolid};
+  color: ${designTokens.colorSolid};
 
   :hover {
-    background-color: ${customProperties.colorNeutral90};
+    ${(props) =>
+      `background-color: ${
+        props.theme === 'test'
+          ? designTokens.colorNeutral95
+          : designTokens.colorNeutral90
```

Comment:
Maybe we could use a design token here. Something like this:
```suggestion
      `background-color: ${appKitDesignTokens.backgroundColorForUserSettingsMenuItem}`
```

Reply:
Since we need to use the hook anyway, I thought it's easier to use if-else logic instead of creating new tokens as it's only for the migration phase.

Do you think it's fine or do you prefer to use design tokens?

Reply:
I mean, that's the purpose of the tokens but I can see your point.
For sure we can use the current option or even the new `themedValue` helper from the `useTheme` hook to make it simpler, however I think I would have used a token for consistency.